dist/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
+    <title>Card</title>
+  </head>
+  <body>
+    <main>
+      <section class="center">
+        <!-- Circles-->
+        <article class="background">
+          <img class="back1" src="images/bg-pattern-top.svg" alt="" />
+          <img class="back2" src="images/bg-pattern-bottom.svg" alt="" />
+        </article>
+
+        <!-- Card -->
+        <article class="card">
```

Comment:
Neka samo Card bude Article, ostalo moze kao obican div. Section imas jedan i to je OK.